src/pybind/mgr/dashboard/frontend/package-lock.json

```diff
@@ -468,7 +468,7 @@
         },
         "minimist": {
           "version": "1.2.0",
-          "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
+          "resolved": "http://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```

Comment:
Please remove this file from the commit.

Reply:
hi, I am not able to remove this file from the commit .How should I do that ?

Reply:
1. git checkout src/pybind/mgr/dashboard/frontend/package-lock.json 
2. git commit
3. git rebase
4. git push -f

Reply:
hi i have checked out the file and committed it again. However I am not able to remove this entire file from the already pushed commit.How can I do that ?. I have returned the file to the original file but the file is still there in the commit.

Reply:
it seems @joscollin 1st step is missing the master branch. I was able to do it with: `git checkout master frontend/package-lock.json`.
After you commit the file you can just squash all your commits and the file should be removed.

Reply:
right, thanks @tspmelo